pages/moderators/reviews/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from "react";
+import PermissionsDashboard from "~/components/PeerReviews/PeerReviewRequestDashboard";
+import ContentPage from "~/components/ContentPage/ContentPage";
+import SideColumn from "~/components/Home/SideColumn";
+import ModeratorDashboardSidebar from "~/components/shared/ModeratorDashboardSidebar";
+
+export default function ReviewsIndex(): ReactElement<
+  typeof PermissionsDashboard
+> | null {
```

Comment:
Wouldn't return type be like below? surpised yarn build didn't throw (?) :o 
```suggestion
export default function ReviewsIndex(): ReactElement<
  typeof ContentPage
> {
```

Reply:
ah yes, you're right. Not sure why I have it there. thanks!